adb_shell/adb_device.py

```diff
@@ -860,7 +861,7 @@ def _read_until_close(self, adb_info):
         while True:
             cmd, data = self._read_until([constants.CLSE, constants.WRTE], adb_info)
 
-            if cmd == constants.CLSE:
+            if bytes(cmd) == constants.CLSE:
```

Comment:
This isn't needed, since `_read_until` returns bytes (because `_read` returns bytes).

Reply:
Changed in https://github.com/Halastra/adb_shell/pull/5